### ISSUE TITLE
[Typo] Remove trailing forward slash from BASH path

### ISSUE THREAD

**Issue by habere-et-dispertire** [Raku/doc#4091]
The [install from source instructions](https://rakudo.org/star/source) under Linux/MacOS reads in part:

```
echo "export PATH=$(pwd)/bin/:...
```

This should perhaps have the trailing forward slash removed from the first path:

```
echo "export PATH=$(pwd)/bin:...
```

Otherwise you get something confusing (if harmless?) like:

```
$ which raku
~/rakudo/bin//raku
```

**Comment by coke** [comment]
I agree with this suggestion, but this ticket should be in the rakudo.org website repo at https://github.com/rakudo/rakudo.org.

(It's an easy enough change so perhaps someone can fix the issue and close it here)

**Comment by coke** [comment]
(Sorry, I see you were also working on something in a fork!)

**Comment by habere-et-dispertire** [comment]
No problem. I am unfamiliar with this source.
You seem to have found two relevant files, so I'll close my pull request -- thank you @coke
